SingularityUI/app/components/requestDetail/header/RequestTitle.jsx

```diff
@@ -39,13 +40,22 @@ const RequestTitle = ({requestId, requestAPI}) => {
     );
   }
 
+  const requestIdToDisplay = Utils.maybe(requestAPI, ['data', 'request', 'id']) || requestId;
```

Comment:
Honestly, I don't know. I copied it from what was there before (see line 48 on the other side of the diff), but I can't justify it.
I'll delete it - `requestId` is marked as required in propTypes after all.